src/lib.rs

```diff
@@ -183,10 +183,14 @@
 //! # Ok(())
 //! # }
 //! ```
+//!
+//! [`pkcs8::DecodePublicKey`]: https://docs.rs/pkcs8/latest/pkcs8/trait.DecodePublicKey.html
+//! [`pkcs8::EncodePublicKey`]: https://docs.rs/pkcs8/latest/pkcs8/trait.EncodePublicKey.html
```

Comment:
For some reason rustdoc is failing to link these. Not sure why, but this at least works around the bug.

See: https://docs.rs/rsa/latest/rsa/#pkcs8-rsa-key-encoding

Reply:
Strange. This works for me with the local cargo doc.